src/extract.ts

```diff
@@ -474,10 +466,10 @@ function extractCatch2Result(output: string): BenchmarkResult[] {
 
 function extractCustomBenchmarkResult(output: string): BenchmarkResult[] {
     try {
-        const json: CustomBenchmarkJson = JSON.parse(output);
-        return json.benchmarks.map(bench => {
+        const json: [BenchmarkResult] = JSON.parse(output);
+        return json.map(bench => {
             const { name, value, unit } = bench;
-            return { name, value, unit, range: undefined, extra: undefined };
+            return { name, value, unit };
```

Comment:
I was tempted to drop the intermediate `const` destructuring . Even though we are saying `json` will be `[BenchmarkResult]` I think we could technically have other fields (or maybe even not have the specified fields). It seems like the `try` / `catch` with the destructure is probably safest.